app/controllers/distributions_controller.rb

```diff
@@ -130,6 +130,16 @@ def pick_ups
     @pick_ups = current_organization.distributions
   end
 
+  def picked_up
+    if Distribution.find_by(id: params['id'])&.scheduled? && Distribution.find_by(id: params['id'])&.complete!
```

Comment:
This is a deferrable change, but in this case, you'd probably want to just combine these, like: 
```suggestion
if Distribution.scheduled.find(params['id'])&.complete!
```
(`:find` accepts the `:id` as the default argument), as well). The main reason to do this is that it would reduce the number of queries by half -- as it stands, we're hitting the database twice. (here's the docs: https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html -- `enum` magically creates scopes)

We can fix it later, NBD. I don't want to hold up this PR any longer :)